### PR TITLE
feat: add consul module

### PR DESCRIPTION
**Pull request by ruslansennov**
HashiCorp Consul
https://www.consul.io/

### CODE REVIEW

File: src/consul/mod.rs [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+use testcontainers::core::WaitFor;
+use testcontainers::Image;
+
+const DEFAULT_IMAGE_NAME: &str = "hashicorp/consul";
+const DEFAULT_IMAGE_TAG: &str = "1.16.1";
+const CONSUL_LOCAL_CONFIG: &str = "CONSUL_LOCAL_CONFIG";
+
+#[derive(Debug)]
+pub struct Consul {
```
Comment (DDtKey) [inline src/consul/mod.rs:11]: Could you add a documentation? We ask this for all new modules

Example https://github.com/testcontainers/testcontainers-rs-modules-community/blob/0963b03da864ae812ce0629917e9de9b69b6c335/src/rabbitmq/mod.rs#L6-L29

**Review by DDtKey — COMMENTED** [review]
Thanks for the contribution!

File: src/consul/mod.rs (hunk 1 above)
Comment (ruslansennov) [reply src/consul/mod.rs:30]: done

**Comment by ruslansennov** [comment]
@DDtKey should I do anything else?

**Review by DDtKey — APPROVED** [review]

**Comment by DDtKey** [comment]
No, that's alright. I changed PR name to match rules of conventional commits

Thank you! 👍

**Pull request merged**